zq2/docs/nodes/validatormonitoring.md

```diff
@@ -0,0 +1,79 @@
+---
+id: nodes/validatormonitoring
+title: Validator Monitoring 
+  - Node 
+  - Validator 
+  - Monitor 
+description: Validator Monitoring 
+---
+
+# Listing in the Network Monitoring Dashboard 
+
+## Overview
+
+This guide explains how to run the Docker container that integrates your node in the public network monitoring dashboard `https://stats.zq2-prototestnet.zilliqa.com/` or `https://stats.zq2-protomainnet.zilliqa.com/`.
+
+## Prerequisites
+* Docker installed on your system
+* Network access to the required endpoints
+
+### Docker Command
+
+```
+docker run -ti --platform linux/x86_64 \
+    --net=host \
+    -e RPC_HOST="localhost" \
+    -e RPC_PORT="4202" \
+    -e LISTENING_PORT="3333" \
+    -e INSTANCE_NAME="node-name" \
+    -e CONTACT_DETAILS="contact@example.com" \
+    -e WS_SERVER="ws://stats.example.com" \
+    -e WS_SECRET="<secret-value>" \
+    -e VERBOSITY="2" \
+    asia-docker.pkg.dev/prj-p-devops-services-tvwmrf63/zilliqa-public/eth-net-intelligence-api:v0.0.1
```

Comment:
latest version is v0.0.2

Reply:
Hi @frankmeds , please push the `v0.0.2` image in the registry anytime.